glossary.asciidoc

```diff
@@ -308,7 +304,7 @@ Noise_XK::
     The template of the Noise protocol framework to establish an authenticated and encrypted communication channel between two peers of the Lightning Network.
     X means that no public key needs to be known from the initiator of the connection.
     K means that the public key of the receiver needs to be known.
-    More particular (from: http://www.noiseprotocol.org/noise.html) the protocol enables encryption to a known recipient and strong forward secrecy. This payload is encrypted based on an ephemeral-ephemeral DH as well as an ephemeral-static DH with the recipient's static key pair. Assuming the ephemeral private keys are secure, and the recipient is not being actively impersonated by an attacker that has stolen its static private key, this payload cannot be decrypted. Sender authentication resistant to key-compromise impersonation (KCI). The sender authentication is based on an ephemeral-static DH ("es" or "se") between the sender's static key pair and the recipient's ephemeral key pair. Assuming the corresponding private keys are secure, this authentication cannot be forged.
+    More particularly (from: http://www.noiseprotocol.org/noise.html), the protocol enables encryption to a known recipient and strong forward secrecy. This payload is encrypted based on an ephemeral-ephemeral DH as well as an ephemeral-static DH with the recipient's static key pair. Assuming the ephemeral private keys are secure, and the recipient is not being actively impersonated by an attacker that has stolen its static private key, this payload cannot be decrypted. Sender authentication is resistant to key-compromise impersonation (KCI). The sender authentication is based on an ephemeral-static DH ("es" or "se") between the sender's static key pair and the recipient's ephemeral key pair. Assuming the corresponding private keys are secure, this authentication cannot be forged.
```

Comment:
Maybe since we're changing this, it would be worth to refactor the first sentence (maybe it's just me, but I don't really like "more particularly") with something like this: "In particular, the protocol (from: http://www.noiseprotocol.org/noise.html) enables encryption and strong forward secrecy to a known recipient."

Reply:
Agreed.  It is the "more + particular" of the original that is the issue for me (using an adjective instead of an adverb).

Reply:
so can we change this to the suggestion from @GiovanniZotta ?

Reply:
Yes.